sdks/csharp/adtIngestFunctionSample.cs

```diff
@@ -22,7 +22,7 @@ public class Function1
         private static readonly string adtInstanceUrl = Environment.GetEnvironmentVariable("ADT_SERVICE_URL");
         // </ADT_service_URL>
         // <HTTP_client>
-        private static readonly HttpClient httpClient = new HttpClient();
+        private static readonly HttpClient singletonHttpClientInstance = new HttpClient();
```

Comment:
The reason for creating an HttpClient we pass in is to prevent socket exhaustion. I figured changing the name of this implies that a bit better. If this were non-sample code, I'd include a comment with a link to the known issue, but I'm assuming we don't want that in the sample.